src/UserAction/ExecutorFactory.php

```diff
@@ -202,7 +202,7 @@ protected function createActionTrigger(UserAction $action, string $type = null):
             $seed = $viewType['default'];
         }
 
-        $seed = is_array($seed) && is_callable($seed) ? call_user_func($seed, $action, $type) : $seed;
+        $seed = is_array($seed) && is_callable($seed) ? $seed($action, $type) : $seed;
```

Comment:
@ibelar can you please fix the seed creation above? For some reasons, array seed can not be invoked as expected not mentioning, in theory, array seed can callable (`is_callable($seed) === true`), but intended to be standard seed - https://3v4l.org/mMgpT